src/subprocess.rs

```diff
@@ -25,10 +51,11 @@ impl std::error::Error for ExitStatusError {}
 impl std::fmt::Display for ExitStatusError {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let _ = write!(f, "subprocess exited with status: ");
```

Comment:
This is extremely nitty but `status: signal 7` reads a wee bit oddly to me.  How would you feel about e.g. "subprocess exited with exit code 5", "subprocess exited with signal 7", "subprocess exited with unknown status"?  Something like

```
let _ = write!(f, "subprocess exited with ");
match self {
    Self::ExitCode(code) => write!(f, "exit code {code}"),
    Self::Signal(signal) => write!(f, "signal {signal}"),
    Self::ExitCode(code) => write!(f, "unknown status"),
}
```

Definitely not a blocker though and there may be others who prefer to leave as is!